content/ecosystem/distributions/distributions.toml

```diff
@@ -0,0 +1,52 @@
+[[distributions]]
+name = "matrix-docker-ansible-deploy"
+description = "🐳 Matrix (An open network for secure, decentralized communication) server setup using Ansible and Docker"
+vendor = "Splanteev"
+maturity = "Stable"
+frameworks = ["Ansible", "Docker"]
+licence = "AGPL-3.0"
+repository = "https://github.com/spantaleev/matrix-docker-ansible-deploy"
+room = "#matrix-docker-ansible-deploy:devture.com"
+support_level = "Community"
+
+[[distributions]]
+name = "Ananace Helm Charts"
+description = "Kubernetes applications packaged into helm charts"
+vendor = "Ananace"
+maturity = "Stable"
+frameworks = ["Helm", "Kubernetes"]
+licence = "Apache-2.0"
+repository = "https://gitlab.com/ananace/charts"
+room = "#matrix-on-kubernetes:fiksel.info"
+support_level = "Community"
+
+[[distributions]]
+name = "Element Server Suite"
+description = "Element Server Suite (ESS) is a professional backend for Matrix-based communications that supports self-hosted and fully managed deployments."
+vendor = "Element"
+maturity = "Stable"
+frameworks = ["Kubernetes"]
+licence = "Proprietary"
+repository = "https://element.io/server-suite"
+support_level = "Commercial"
+
+[[distributions]]
+name = "synapse-operator"
+description = "The Synapse operator offers a convenient way to deploy and manage a Synapse server. It was built with operator-sdk."
+vendor = "Red Hat"
+maturity = "Stable"
+frameworks = ["Kubernetes"]
+licence = "Apache-2.0"
+repository = "https://github.com/opdev/synapse-operator"
+support_level = "Community"
+
+
+[[distributions]]
+name = "BundesMessenger"
```

Comment:
Some distros like BUM will have something special about them that makes them not exactly matrix standard (compatible), even though the distro still meets the requirement in the page summary. This should be pointed out prominently, or even qualify them for a different category.

In the case of BUM i think there might be something like antivirus or allowlisting etc involved which requires specialized clients(?). so it's a weird edgecase.